endpoints-control/src/main/java/com/google/api/control/ControlFilter.java

```diff
@@ -263,30 +268,48 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 
       // 'Send' a report, end the latency timer to collect correct overhead and backend latencies
       timer.end();
-      ReportRequest reportRequest =
-          createReportRequest(info, checkInfo, appInfo, ConfigFilter.getReportRule(request), timer);
-      if (log.isLoggable(Level.FINEST)) {
-        log.log(Level.FINEST, String.format("sending an error report request %s", reportRequest));
-      }
-      client.report(reportRequest);
-
+      boolean trickle = false;
```

Comment:
My guess is the trickle stuff is supposed to deal with an overloaded service we report into, but I'm not sure what and why. Consider adding some explanation here.